hs_core/hydroshare/hs_bagit.py

```diff
@@ -205,6 +205,7 @@ def create_bag_files(resource):
     # and map xml documents
     if resource.resource_type == "CompositeResource":
         resource.create_aggregation_xml_documents()
+        resource.create_model_program_meta_json_schema_files()
```

Comment:
why can't this just be a part of creating the aggregation xml documents in the method above?  I can't think of a time we wouldn't want to tie those two functions together.